Fails to build

```
error[E0107]: struct takes 0 generic arguments but 1 generic argument was supplied
   --> /home/ubuntu/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-extract-0.2.2/src/lib.rs:169:45
    |
169 | fn set_unix_mode<R: Read>(file: &zip::read::ZipFile<R>, outpath: &Path) -> io::Result<()> {
    |                                             ^^^^^^^ - help: remove the unnecessary generic argument
    |                                             |
    |                                             expected 0 generic arguments
    |
note: struct defined here, with 0 generic parameters
   --> /home/ubuntu/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-2.5.0/src/read.rs:252:12
    |
252 | pub struct ZipFile<'a> {
    |            ^^^^^^^

   Compiling calamine v0.26.1 (https://github.com/prophittcorey/calamine?branch=prophittcorey%2Ffix-zip-dependency#2fc1a8df)
error[E0283]: type annotations needed
   --> /home/ubuntu/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-extract-0.2.2/src/lib.rs:132:9
    |
132 |         set_unix_mode(&file, &outpath)?;
    |         ^^^^^^^^^^^^^ cannot infer type of the type parameter `R` declared on the function `set_unix_mode`
    |
    = note: cannot satisfy `_: std::io::Read`
note: required by a bound in `set_unix_mode`
   --> /home/ubuntu/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zip-extract-0.2.2/src/lib.rs:169:21
    |
169 | fn set_unix_mode<R: Read>(file: &zip::read::ZipFile<R>, outpath: &Path) -> io::Result<()> {
    |                     ^^^^ required by this bound in `set_unix_mode`
help: consider specifying the generic argument
    |
132 |         set_unix_mode::<R>(&file, &outpath)?;
    |                      +++++

Some errors have detailed explanations: E0107, E0283.
For more information about an error, try `rustc --explain E0107`.
error: could not compile `zip-extract` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ default = ["zip/default"]
 
 [dependencies]
 log = "0.4"
-zip = { version = "2", default-features = false }
+zip = { version = ">=2.6", default-features = false }
 thiserror = "2.0"
 
 [dev-dependencies]

